parse.go

```diff
@@ -41,24 +41,29 @@ func parseFile(fset *token.FileSet, filePath, template string) (af *ast.File, mo
 		case *ast.GenDecl:
 			switch typ.Tok {
 			case token.CONST, token.VAR:
+				var parenFlag bool
 				if !(typ.Lparen == token.NoPos && typ.Rparen == token.NoPos) {
-					// if there's a () add comment for each sub entry
-					for _, spec := range typ.Specs {
-						vs := spec.(*ast.ValueSpec)
-						if !vs.Names[0].IsExported() {
-							continue
+					// if there's a () and parenComment is false, add comment for each sub entry
+					if !*parenComment {
```

Comment:
Seems this invert the logic, we want to add comment for each sub entry if parentComment is true.

Reply:
If parenComment is true, add comment for each sub entry or add comment for the whole parens
Default is false

Reply:
ok.Default is false to add one comment to the whole parens, sorry for this careless mistake